### PR TITLE
Allow passing raw html to Truncate

### PR DESCRIPTION
**Pull request by akofink**
For https://github.com/RedHatInsights/compliance-frontend/pull/103

### CODE REVIEW

File: src/PresentationalComponents/Truncate/Truncate.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ class Truncate extends React.Component {
         });
     }
 
+    dangerousHtml(text) {
+        return { __html: text };
```
Comment (ryelo) [inline src/PresentationalComponents/Truncate/Truncate.js]: Do we want to sanitize this html before setting it?

If so, we may want to use something like: https://github.com/cure53/DOMPurify

Comment (akofink) [reply src/PresentationalComponents/Truncate/Truncate.js]: Sounds like a good idea. I'll look into it. Thanks for the review!

**Comment by akofink** [comment]
Updated with DOMPurify

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "c3": "^0.6.7",
     "classnames": "^2.2.5",
     "d3": "^5.7.0",
+    "dompurify": "^1.0",
```
Comment (karelhala) [inline package.json]: Advisor is using sanitize-html [1] please use it as well so we don't have two packages for same thing.

[1] https://github.com/RedHatInsights/insights-frontend-components/blob/master/src/SmartComponents/Inventory/applications/advisor/ExpandableRulesCard.js#L7

**Comment by akofink** [comment]
Updated with sanitize-html

**Pull request merged**